CMakeLists.txt

```diff
@@ -77,7 +77,7 @@ cmake_dependent_option(IREE_ENABLE_EMITC "Enables MLIR EmitC dependencies." ON $
 
 option(IREE_HAL_DRIVER_DEFAULTS "Sets the default value for all runtime HAL drivers" ON)
 # CUDA support must be explicitly enabled.
-set(IREE_HAL_DRIVER_CUDA_DEFAULT OFF)
+option(IREE_HAL_DRIVER_CUDA_DEFAULT OFF)
```

Comment:
The user-settable option is below this - `IREE_HAL_DRIVER_CUDA`. This is just the default value for that option.

See https://google.github.io/iree/deployment-configurations/gpu-cuda-rocm/#build-runtime-from-source

Reply:
Before the great Cuda build debacle, the default for Cuda was different depending on platform (similar to how Vulkan is below). Now it just always defaults to `OFF`. That could be inlined :shrug: